tests/schedulers/test_schedulers.py

```diff
@@ -238,6 +238,12 @@ def get_scheduler_config(self):
 
     def dummy_model(self):
         def model(sample, t, *args):
+            # if t is a tensor, match the number of dimensions of sample
+            if isinstance(t, torch.Tensor):
+                num_dims = len(sample.shape)
+                # pad t with 1s to match num_dims
+                t = t.reshape(-1, *(1,) * (num_dims - 1))
+
```

Comment:
Could I get a brief explanation on why was this needed?

Reply:
Also curious here

Reply:
Typically timestep `t` is a scalar, and `sample` has shape e.g. (batch, 3, 8, 8).
For parallel sampling, `t` has shape (batch,) and `sample` has shape (batch, 3, 8, 8).

When running net.forward(sample, t, ...) in with the typical unet, this runs fine.